test/fixtures/incorrect.js

```diff
@@ -123,17 +131,16 @@ try {
   noop();
 } catch (e) {}
 
-// `no-labels`.
+// `no-labels` and `no-unused-labels`.
```

Comment:
We're not using `no-unused-labels` in our list of rules. Should we ignore this, since it's a rule that gets redundant by `no-labels` anyway?